eslint-rules/index.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable @wordpress/no-unused-vars-before-return */
```

Comment:
My initial feeling is I'd prefer to avoid adding another dev tooling-related top level directory, but also not sure where else to put it. Any thoughts @tofumatt @aaemnnosttv?

Reply:
Honestly, something like `packages/eslint-plugin/` seems perhaps the most appropriate. I feel like we are getting to the point where we should start thinking about publishing packages _eventually_, but for now they can be entirely internal and we can require them via our `package.json` with `file:packages/eslint-plugin` [just like Gutenberg does](https://github.com/WordPress/gutenberg/blob/99bff621c853cb65fe99f168afb6ea861af1827a/package.json#L123).

Reply:
I like that. @tofumatt Can you move it into a `packages` folder?